CHANGELOG

```diff
@@ -1,30 +1,74 @@
 [REFINE]        Mac: Make engine compatible for 32bit Mac.
 [REFINE]        Audio: AudioEngine on Linux replace the original SimpleAudioEngine with a new version of FMOD, now AudioEngine support all platforms!
+[REFINE]        IOS: Add virtual keyword for some render related function.
+[REFINE]        UI: Fixes boring deprecated warning in HttpRequest.
+[REFINE]        Network: Fix Downloader bug on iOS & Android platform.
+[REFINE]        Studio: Fix deprecation warning in SkeletonRenderer.
+[REFINE]        JS: Add js test case for fix, improve template.
+[REFINE]        Network: Permit http access to cocos2d-x.org in test projects on iOS.
+[REFINE]        Network: Crash when removing a remotely downloaded image from texture cache in js-binding.
+[REFINE]        Win10: WinRT project update version to v3.10.
+[REFINE]        Console: Add quiet option for Cocos Toolkit.
+[REFINE]        JS: New GC model for js-binding.
+[REFINE]        Doc: Fix typos in documentation and comments.
+[REFINE]        UI: update controlButton size calculate with new Scale9Sprite logic.
+[REFINE]        Win10: Added missing _USRJSSTATIC preprocessor define for ARM builds.
+[REFINE]        JS: Added ccvector_to / ccmap_to converted to new js-binding API.
+[REFINE]        UI: Slider misprint fix.
+[REFINE]        UI: Scale9Sprite cropped sprite frames support.
 
 [FIX]           Core: Fix premultiplyAlpha for mipmaps and compressed textures.
 [FIX]           UI: Fix Scale9sprite rendering error when content size smaller than the sum of leftInset and rightInset.
 [FIX]           Win32: Fix EditBox crash when removing an EditBox in a scheduler.
 [FIX]           Android: Fix cannot add view to mFrameLayout when extends Cocos2dxActivity.
-[FIX]           2D: Fixed actionNode position error bug.
+[FIX]           2D: Fixed actionNode set at wrong position bug.
 [FIX]           3D: Fix the  movement of PUParticle lags one frame.
 [FIX]           UI: Fix the wront argument of setPlaceholderFontName in EditBox.
 [FIX]           UI: Fix EditBox editBoxEditingDidEnd may use the original text after change the text of EditBox in user script.
 [FIX]           Audio: Fix `FinishCallback` never be called in Windows.
-[FIX]           UI:  Fix Layout stencil clipping nested with Clipping Node rendering issue.
-[FIX]           UI:  Keyboard doesn't hide when click the screen outside of EditBox on iOS platform.
+[FIX]           UI: Fix Layout stencil clipping nested with Clipping Node rendering issue.
+[FIX]           UI: Keyboard doesn't hide when click the screen outside of EditBox on iOS platform.
+[FIX]           UI: Fix a fatal bug in EditBox implement on Windows platform.
+[FIX]           UI: Fix edit box setPlaceholderFontName and scale font size issue.
+[FIX]           Core: Fix memory leak when initWithImage() failed.
+[FIX]           Network: CCDownloader on iOS is broken in v3.9 js-binding.
+[FIX]           UI: Fix label clamp when scaled with bmfont.
```

Comment:
`[FIX]           UI: Fix label clamp when scaled with bmfont.` This one is not needed.